src/scripting/game_lua_kernel.cpp

```diff
@@ -382,7 +382,11 @@ int game_lua_kernel::impl_add_movement(lua_State* L)
 		lua_pushinteger(L, temp.wml_y());
 	}
 	lua_pushboolean(L, true);
-	lua_call(L, 4, 0);
+	if(lua_isnoneornil(L, 4))
+		lua_pushnil(L);
+	else
+		lua_pushboolean(L, luaW_toboolean(L, 4));
```

Comment:
Please either:

1. Add braces.

```suggestion
	if(lua_isnoneornil(L, 4)) {
		lua_pushnil(L);
	} else {
		lua_pushboolean(L, luaW_toboolean(L, 4));
	}
```

2. Make them one-liners.

```suggestion
	if(lua_isnoneornil(L, 4)) lua_pushnil(L);
	else lua_pushboolean(L, luaW_toboolean(L, 4));
```

(Note: While I'm fine with either way, I believe most people here at Wesnoth prefer the first option.)